website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -5,59 +5,75 @@ description: >-
+* You’ll see a [list of Datasources](/reference/datasources/) that Appsmith can connect to.
+* Choose a **Database/API**.
+* Provide the **configuration details** required to connect to your database. You may need to contact your database administrator to [whitelist appsmith cloud](/learning-and-resources/how-to-guides/aws-whitelist).
+* Click **Test** to verify that Appsmith can connect to your database using the details you provided.
+* Rename and save your datasource.
 
-:::tip
-Databases configured within one page of an app can be accessed by developers anywhere within the workspace.
+Once your datasource has been added successfully, a success pop-up will appear at the top. 
+
+### Sample database
+
+
+Appsmith offer dummy database that you can use to experiment with the platform before adding your real data. You can connect to two sample datasets:
+
+1. **Movies:** This is a sample [MongoDB](/reference/datasources/querying-mongodb/) database with a collection called movies.
+2. **Users:** This is a sample [PostgreSQL](/reference/datasources/querying-postgres) database with a table called users.
+
+:::note
+The **data** in the **mock database** gets **reset** every **24 hours** and is **common** for **all users** so it may contain **some unexpected values**.
 :::
 
-## General Notes
+To better understand how to use the sample database, lets look at an example:
 
-Appsmith creates a new connection pool with the database server when you first connect the database server to your app. All subsequent queries executed by Appsmith against your database then re-use this connection to ensure that at run-time your queries are executed quickly. In the case that an idle connection is closed by the database server, Appsmith creates a new connection while executing the next query.
+<VideoEmbed host="youtube" videoId="l_MNNa9mg0w" title="Using A Sample Database " caption="How to use sample database | Example"/>
 
-For certain plugins like [PostgreSQL](../../reference/datasources/querying-postgres.md), and [MySQL](../../reference/datasources/querying-mysql.md), Appsmith creates and maintains a connection pool because multiple queries cannot be executed against a single connection.
+* Click on **+** next to Datasources.
+* Select the **user** datasource.
+* Now, click the **+** icon next to the **queries/js** and choose the user datasource.
+* Add your code in the body section and **run** your query. For example:
+```js
+SELECT * FROM users;
+```
+The preceding query gets all the users data. You can use a [table widget](/reference/widgets/table) to display your data. 
 
-### Concurrent Queries
+:::info
+Appsmith applications are **secure-by-default**. All sensitive credentials, such as database credentials, are encrypted with [**AES-256 encryption**](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard). You can read more about security in this [document](/product/security#security-measures-within-appsmith). 
+:::
+
+
```

Comment:
I noticed that the General Notes and Concurrent Queries sections are missing. Please keep them. We shouldn't delete any sections of information during rehaul. We can restructure them by moving them to the right location and rewrite them to give more clarity, but we shouldn't cut out any information